### PR TITLE
[RPM] Update foreman remote execution to 1.4.6

### PR DESCRIPTION
**Pull request by adamruzicka**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
I tend to forget this, but could you add changelog entries? https://github.com/theforeman/foreman-packaging/pull/2259 should update the documentation and tooling to match the current workflows.

While you're at it, could you squash the dependency update commit into the REX update commit?

**Comment by adamruzicka** [comment]
No problem, sorry for forgetting it (again)

**Comment by ekohl** [comment]
I still forget it most of the time. We need a check in Jenkins for this and get that update merged so we have tools in place that do the right thing.

**Comment by adamruzicka** [comment]
@ekohl anything else to do here?

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}rubygems
 Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.26
 Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
-Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.9.5
+Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.12.0
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: ```~> 0.12``` means ```>= 0.12.0``` and ```< 1.0.0```

**Comment by adamruzicka** [comment]
anything else?

**Pull request merged**